unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Protocol/Protocol.cs

```diff
@@ -31,6 +31,12 @@ public static class ComponentNameLiterals
         public const string OnPointerHoverExit = "pointerHoverExit";
     }
 
+    public enum ECS7_CLASS_ID
+    {
+        BOX_SHAPE = 1016,
+        SPHERE_SHAPE = 1017,
+    }
+    
```

Comment:
could we avoid adding this at this point, I think is not relevant at the moment and I'm not sure if `Protocol.cs` should be the place to put this kind of enum

Reply:
We need to specify somewhere the ID of the components for the rendereable. We can't use the same ID as the legacy ones so we need to store this information somewhere, we can move it to ProtocolV2 if you want